--- fileio.F	2014-11-04 09:26:26.000000000 -0600
+++ fileio_new.F	2015-02-10 07:04:13.000000000 -0600
@@ -1027,6 +1027,7 @@
 ! work arrays
       RGRID,  ALLOCATABLE ::  CWORK(:)
       REAL(q),ALLOCATABLE ::  WORK (:)
+      REAL(q) chsum !karol 
       INTEGER NALLOC,NZ, NWRITE, NWRITTEN
       CHARACTER (40) FORM
       INTEGER ISTAT
@@ -1066,17 +1067,16 @@
       NWRITTEN=0
       DO NZ=1,GRIDC%NGZ
          CALL MRG_GRID_RL_PLANE(GRIDC, WORK, CWORK, NZ)
-         io_begin
+
+         chsum = 0.0_q
          DO N=1,NALLOC
-            NWRITTEN=NWRITTEN+1
-            IF ( MOD(NWRITTEN,NWRITE)==0 ) THEN
-               WRITE(IU,FORM) WORK(N)
-            ELSE
-               WRITE(IU,FORM,ADVANCE='NO') WORK(N)
-            ENDIF
+           NWRITTEN=NWRITTEN+1
+           chsum = chsum + WORK(N)
          ENDDO
-         io_end
+
+         do_io WRITE(IU,FORM) chsum
       ENDDO
+
       IF ( MOD(NWRITTEN,NWRITE)/=0 ) WRITE(IU,*)' '
 
       DEALLOCATE(CWORK,WORK)
